cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/worker/TaskWorkerTestUtil.java

```diff
@@ -58,7 +58,7 @@ public void failed(Service.State from, Throwable failure) {
    *
    * @param future {@link CompletableFuture<Service.State>}
    */
-  static void waitForServiceCompletion(CompletableFuture<Service.State> future) {
+  public static void waitForServiceCompletion(CompletableFuture<Service.State> future) {
```

Comment:
You've added SystemWorkerTestUtil.java. Why do we need to also use TaskWorkerTestUtil?
Either merge the two into one class, or completely separate them.

Reply:
Seperated them.